packages/data/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  finish,
+  retrieveChild,
+  retrievePastSessions,
+  retrieveResponse,
+  retrieveStudy,
+  updateResponse,
+} from "./api";
+
+import s3 from "./s3";
+import { Child, PastSession, Response, Study } from "./types";
+
+declare global {
+  interface Window {
+    chs: {
+      study: Study;
+      child: Child;
+      pastSessions: PastSession[];
+      response: Response;
+    };
+  }
+}
+
+/**
+ * Load data from API that would be used by researchers and jsPsych.
+ *
+ * @param response_uuid - Response UUID.
+ */
+const load = async (response_uuid: string) => {
```

Comment:
So this doesn't return the CHS object, right? 
In that case, maybe we should note in the description that it attaches the following to the window: 'chs' object with properties: 
- 'study' (object)
- 'child' (object)
- 'pastSessions' (array of objects, where each object is a past session/response)
- 'response' (object)